superbench/benchmarks/micro_benchmarks/cudnn_test/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
For the folder name, whether do we change it as cublas_function? It's very strange that one is cudnn_test and the other is cublas_function

Reply:
renamed